packages/smart-wallet/src/walletFactory.js

```diff
@@ -27,10 +27,14 @@ export const meta = {
     { storageNode: M.eref(M.remotable('StorageNode')) },
     { walletBridgeManager: M.eref(M.remotable('walletBridgeManager')) },
   ),
-  upgradability: 'canUpgrade',
+  // UNTIL this compat stuff lands???@@@
+  // upgradability: 'canUpgrade',
```

Comment:
can/should I turn this back on, @turadg ?

Reply:
let's not muck with `meta` for now. it was motivated to simplify new contracts.